Assets/MixedRealityToolkit.Tools/ExtensionServiceCreator/Templates/ExtensionScriptTemplate.txt

```diff
@@ -6,20 +6,18 @@ namespace #NAMESPACE#
 	[MixedRealityExtensionService(#SUPPORTED_PLATFORMS_PARAM#)]
 	public class #SERVICE_NAME# : BaseExtensionService, #INTERFACE_NAME#, IMixedRealityExtensionService
 	{
-		private #PROFILE_NAME# #PROFILE_FIELD_NAME#;
-
-		public #SERVICE_NAME#(string name,  uint priority,  BaseMixedRealityProfile profile) : base(name, priority, profile) 
-		{
-			#PROFILE_FIELD_NAME# = (#PROFILE_NAME#)profile;
-		}
-
+#SERVICE_CONSTRUCTOR#
```

Comment:
I think preserving this blank line would be good for when the contents of ExtensionConstructorTemplate are dropped in

```suggestion
#SERVICE_CONSTRUCTOR#

```